askama_derive/Cargo.toml

```diff
@@ -39,6 +39,9 @@ prettyplease = "0.2.20"
 similar = "2.6.0"
 syn = { version = "2.0.3", features = ["full"] }
 
+[build-dependencies]
+autocfg = { version = "1.5.0", optional = true }
```

Comment:
We could omit the `autocfg` dependency and simply generate code that fails is the user opts in to `-F nightly-spans` on `+stable`. Thinking more about it, this might be the better idea, because then we wouldn't need the `build.rs` script either.